### DISCUSSION TITLE
Reducing a superlative/comparative adjective to its "base" form

### DISCUSSION THREAD

**Discussion by scpedicini** [nlp-compromise/nlp_compromise#861] [Q&A]
I noticed that compromise has functions in the compromise-adjectives plugin for conjugating as a comparative/superlative:
`nlp('tasty').adjectives(0).toSuperlative().out()` => "tastiest"

Is it possible to reverse the process, eg:
`nlp('A noble spirit embiggens the smallest man').adjectives(1).terms().json({baseForm: true})`

Something equivalent to that which would produce: "small".

I tried looking around the documentation but couldn't find anything, just want to make sure I'm not missing anything obvious.

**Comment by spencermountain** [comment]
you know, i've never thought of that.
of course we should do that. good idea

**Comment by spencermountain** [comment]
i am working on v14 right now, but can add this once that's ready later this summer
cheers
